scripts/run_nextflow.sh

```diff
@@ -0,0 +1,2 @@
+unzip -o /opt/nextflow/batch.zip
```

Comment:
Unless there's a reason to exclude?

```suggestion
#!/bin/bash

unzip -o /opt/nextflow/batch.zip
```

Reply:
This was no longer needed.  Codedeploy is handing this.